src/overrides/DocumentFormExtensions.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
```

Comment:
Is `src/overrides` a new folder? What is the purpose of it?

Reply:
It is **temporary**. The `index.js` where I was putting all the override implementations started to grow, so I put them all in the ovverides folder. The purpose of it is to disappear and move its components to overlay.